packages/api/src/hooks/useLoginHistory.ts

```diff
@@ -0,0 +1,30 @@
+import { useMemo } from 'react';
+import { getLoginHistoryFormattedData } from '@deriv/utils';
+import useAuthorize from './useAuthorize';
+import useFetch from '../useFetch';
+
+/** A custom hook to retrieve a summary of login history for user.*/
+const useLoginHistory = () => {
+    const { isSuccess } = useAuthorize();
+    const { data, ...rest } = useFetch('login_history', {
+        payload: { limit: 50 },
```

Comment:
This looks like a nice hook, we can make it a bit flexible by providing the option to pass `limit` or payload to `useLoginHistory`